### PR TITLE
fix(AChatForm, AChat): fix `isScrolledBottom` and set max-rows

### PR DESCRIPTION
**Pull request by bludnic**
https://trello.com/c/7jZEG9rt/228-bug-multiple-strings-hide-last-messages

### CODE REVIEW

**Review by adamant-al — APPROVED** [review]

File: src/components/AChat/AChat.vue [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default {
       const nonVisibleClientHeight =
         this.$refs.messages.scrollHeight -
         this.$refs.messages.clientHeight -
-        this.$refs.messages.scrollTop
+        Math.ceil(this.$refs.messages.scrollTop)
```
[the remark this answers is not in the document]
Comment (bludnic) [reply src/components/AChat/AChat.vue:102]: what do you mean? there is no method with the name `scrollTo`

[the remark this answers is not in the document]
Comment (bludnic) [reply src/components/AChat/AChat.vue:102]: The `scrollTo()` method updates the value of `Element.scrollTop`, but we need to round up when getting that value, so the `scrollTo` method is not the right place for rounding.

[the remark this answers is not in the document]
Comment (adamant-al) [reply src/components/AChat/AChat.vue:102]: ✓

**Comment by adamant-al** [comment]
Merge it.

**Pull request merged**